troubleshooting/known_issues/kube_multus_pod_in_ImagePullBackOff.md

```diff
@@ -0,0 +1,34 @@
+# Kube-multus pod is in ImagePullBackOff
+
+## Description
+
+There is a known problem where kube-multus pods may fails to re-start due to an ImagePullBackOff error. The image will need to be re-tagged in nexus and the pod will need to be restarted.
```

Comment:
. . . may fail to re-start . . .

Reply:
As many times as I read that too ... thanks; fixed